client/components/Photos.js

```diff
@@ -38,7 +40,7 @@ const Photos = ({ currentTrip, currentUser }) => {
       <div>
         <Button variant="contained" component="label">
           Upload Photo
-          <input type="file" style={{ display: 'none' }} onChange={fileSelectHandler} />
+          <input type="file" style={{ display: 'none' }} multiple onChange={fileSelectHandler} />
```

Comment:
put this is into a scss file, rather than using in-line styling

Reply:
fixed